### PR TITLE
Jionghao: Added button to switch light/dark modes

### PR DESCRIPTION
**Pull request by Jhwu3**
Implemented button for light/dark modes
and started a footer for social medias
L=ENG-71

### CODE REVIEW

File: js/package.json [hunk 1]
```diff
@@ -21,6 +21,8 @@
     "@mantine/core": "7.10.2",
     "@mantine/hooks": "7.10.2",
     "gsap": "^3.12.5",
+    "@mantinex/mantine-logo": "^1.0.1",
```
Comment (arklian) [inline js/package.json:24]: Can you fix your commit message to be L=ENG-71?

File: js/src/personal/jionghao/Jionghao.page.tsx [hunk 2]
```diff
@@ -1,34 +1,57 @@
...
-        mt="xl"
-      >
-        Hey! I'm Jionghao.
-      </Text>
-    </>
+        <Button onClick={changeMode}>
+          <IconSunHigh size={25} />{' '}
+        </Button>
+      </div>
+      <div>
```
Comment (arklian) [inline js/src/personal/jionghao/Jionghao.page.tsx:52]: Is this div necessary?

File: js/src/personal/jionghao/FooterSocial.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  IconBrandInstagram,
+} from '@tabler/icons-react'
+import { MantineLogo } from '@mantinex/mantine-logo'
+import classes from './FooterSocial.module.css'
+
+export function FooterSocial() {
+  return (
+    <div className={classes.footer}>
+      <Container className={classes.inner}>
+        <MantineLogo size={28} />
```
Comment (arklian) [inline js/src/personal/jionghao/FooterSocial.tsx:14]: Do you actually want the Mantine logo?
If not, we likely shouldn't import the package.

**Comment by arklian** [comment]
Can you add a screenshot?

**Comment by Jhwu3** [comment]
image

File: js/package.json [hunk 4]
```diff
@@ -20,6 +20,7 @@
   "dependencies": {
```
Comment (arklian) [inline js/package.json:20]: `L = ENG-71` -> `L=ENG-71`

You also have a squashed temp to remove from your commit message.

File: js/src/personal/jionghao/Jionghao.page.tsx [hunk 5]
```diff
@@ -1,34 +1,62 @@
...
+        <Title
+          className={styles.title}
+          ta="center"
+          mt={100}
+        >
+          {'Welcome to '}
+          <Text
+            inherit
+            variant="gradient"
+            // component="span"
```
Comment (arklian) [inline js/src/personal/jionghao/Jionghao.page.tsx:32]: nit: Commented out line

**Review by arklian — APPROVED** [review]
Nice!

**Pull request merged**